plugins/scaffolder/src/components/OngoingTask/OngoingTask.tsx

```diff
@@ -223,7 +223,7 @@ function OngoingTaskContent(props: {
   const [, { execute: triggerRetry }] = useAsync(async () => {
     if (taskId) {
       analytics.captureEvent('retried', 'Template has been retried');
-      await scaffolderApi.retry?.(taskId);
+      await scaffolderApi.retry?.({ taskId });
```

Comment:
I think that this is a breaking change to the client which we can't make yet, this needs updating in our wrapping client to take just `taskId`.

There's a bigger piece of work which is to make sure that all the functions of the scaffolder API follow this pattern `({ taskId })` for example, but rather make that change as a bigger breaking change instead.